search/search-tool/tool/pom.xml

```diff
@@ -100,6 +100,11 @@
         <artifactId>commons-lang3</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.elasticsearch</groupId>
```

Comment:
Oh, I see for the test maybe? Maybe it can just have a 

`<scope>test</scope>`